### ISSUE TITLE
Can't get JPEG result image, i always get PNG

### ISSUE THREAD

**Issue by polskiemedia** [CrackerakiUA/ngImgCropFullExtended#92]
I loading JPG file in input
![image](https://cloud.githubusercontent.com/assets/4907660/14015791/e8395666-f1bb-11e5-84e8-13eee7e63fb2.png)

In myCroppedImage i get always PNG
![image](https://cloud.githubusercontent.com/assets/4907660/14015850/25623c6a-f1bc-11e5-8718-0f59f31dd779.png)

even if i set result-image-format="image/jpeg"

**Comment by CrackerakiUA** [comment]
Better give codepen with your setup.

**Comment by VenkatTeja** [comment]
+1
result-image-format="image/jpeg" doesnot work. The resultant datauri is still png. This works in ngcimgcrop. But not working in ngimgcropfullextended. Please look in to it.

**Comment by CrackerakiUA** [comment]
We have mentioned this issue before, you can search on the solved issues.

**Comment by rallu** [comment]
I found reason for this problem. Angular tries to parse image/jpeg as calculation where you divide image variable with jpeg variable. So solution is to force parameter into string.

Like this: `result-image-format="'image/jpeg'"` or you could set variable in controller as a string and then pass it to template.

**Comment by VenkatTeja** [comment]
great!! 

```
On Friday, April 1, 2016 3:19 PM, Honchar Denys <notifications@github.com> wrote:
```

 Closed #92.—

**Comment by CrackerakiUA** [comment]
??
